src/ray/core_worker/core_worker.cc

```diff
@@ -4947,6 +4953,16 @@ void CoreWorker::HandlePlasmaObjectReady(rpc::PlasmaObjectReadyRequest request,
   send_reply_callback(Status::OK(), nullptr, nullptr);
 }
 
+void CoreWorker::HandleCleanUpGPUObject(rpc::CleanUpGPUObjectRequest request,
```

Comment:
```suggestion
void CoreWorker::HandleFreeActorObject(rpc::CleanUpGPUObjectRequest request,
```

Match naming convention for other parts of Ray.

Reply:
Updated https://github.com/ray-project/ray/pull/53911/commits/126015421a85dba7370d62e489c6d3ee51ad5e75